app/components/common/TeamLeaderboardTable.component.js

```diff
@@ -1,52 +1,50 @@
-/* eslint-disable import/no-dynamic-require */
-/* eslint-disable global-require */
-import React from 'react';
-import PropTypes from 'prop-types';
-import { Table, List} from 'semantic-ui-react';
-import { FormattedNumber } from 'react-intl';
-import './style.scss';
+import React from "react";
+import PropTypes from "prop-types";
+import { Table } from "semantic-ui-react";
+import Avatar from "./Avatar.component";
+import "./style.scss";
 
-const TeamLeaderboardTable = ({
-  className, height, teams
-}) => (
-  <div>
-    <List style={{ height: height }} className={className}>
-      <Table celled collapsing basic='very'>
-        <Table.Header>
-          <Table.Row>
-            <Table.HeaderCell>#</Table.HeaderCell>
-            <Table.HeaderCell>Team Name</Table.HeaderCell>
-            <Table.HeaderCell>Distance</Table.HeaderCell>
-            <Table.HeaderCell>Finish Date</Table.HeaderCell>
+const TeamLeaderboardTable = ({ height, teams }) => (
+  <div style={{ height: height }} className={"leaderboard"}>
+    <Table collapsing basic="very" className="main">
+      <Table.Header>
```

Comment:
Should this header be part of the scrollview or only the body? If I scroll I lose the header of the table currently.

Reply:
Good point! It's not clear from the design though, will clarify this with Gavin!